src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -927,11 +928,17 @@ namespace
             QFormLayout* sublayout = create_form_layout();
             layout->addLayout(sublayout);
 
-            QSpinBox* volume_distance_samples =
-                create_integer_input("advanced.volume_distance_samples", 1, 1000, 1);
-            volume_distance_samples->setToolTip(
-                m_params_metadata.get_path("pt.volume_distance_samples.help"));
-            sublayout->addRow("Distance Samples:", volume_distance_samples);
+            QSpinBox* equiangular_distance_samples =
+                create_integer_input("advanced.equiangular_distance_samples", 0, 1000, 1);
```

Comment:
I made it possible to allow using only exponential/equiangular samples. But as we discussed in Slack, we are not going to use separate controls for different distance samples, so I will change it back to 1.